includes/admin/reporting/tables/llms.table.quizzes.php

```diff
@@ -199,7 +199,7 @@ protected function get_data( $key, $data ) {
 						$grade += $attempt->get( 'grade' );
 					}
 
-					$value = round( $grade / $attempts, 3 ) . '%';
+					$value = round( $grade / $attempts, llms_get_reporting_round_precision() ) . '%';
```

Comment:
This change would require a docblock changelog update to me.
Also the file header's `version` tag should be updated as well.